Contributing.md

```diff
@@ -0,0 +1,177 @@
+# Contributing to NovelRT
+
+First off, thank you for taking the time to contribute! We really appreciate it!
+
+The following is a set of guidelines for contributing to NovelRT, which is hosted in the [NovelRT Organization](https://github.com/novelrt) on GitHub. These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+#### Table Of Contents
+
+[Code of Conduct](#code-of-conduct)
+
+[I don't want to read this whole thing, I just have a question!!!](#i-dont-want-to-read-this-whole-thing-i-just-have-a-question)
+
+[What should I know before I get started?](#what-should-i-know-before-i-get-started)
+  * [Design Decisions](#design-decisions)
+
+[How Can I Contribute?](#how-can-i-contribute)
+  * [Reporting Bugs](#reporting-bugs)
+  * [Suggesting Enhancements](#suggesting-enhancements)
+  * [Your First Code Contribution](#your-first-code-contribution)
+  * [Pull Requests](#pull-requests)
+
+[Additional Notes](#additional-notes)
+  * [Issue and Pull Request Labels](#issue-and-pull-request-labels)
+
+## Code of Conduct
+
+This project and everyone participating in it is governed by the [NovelRT Contributor Code of Conduct](CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code. Please report unacceptable behavior to [admin@novelrt.dev](mailto:admin@novelrt.dev) or message one of the maintainers on Discord.
+
+## I don't want to read this whole thing I just have a question!!!
+
+> **Note:** *Please don't file an issue to ask a question.* You'll get faster results by using the resources below.
+
+We have an official message board on Github that you can utilize, or you can reach us on Discord where the community chimes in with helpful advice if you have questions.
+
+* [Github Discussions, NovelRT](https://github.com/novelrt/NovelRT/discussions)
```

Comment:
```suggestion
* [Github Discussions](https://github.com/novelrt/NovelRT/discussions)
```

Maybe...? Unsure. It just read weird to me.

Reply:
Changed in latest